packages/airnode-validator/src/config/config.test.ts

```diff
@@ -642,6 +646,75 @@ describe('chainConfigSchema', () => {
   });
 });
 
+describe('defaultAirnodeRrp', () => {
+  const config = JSON.parse(
+    readFileSync(join(__dirname, '../../test/fixtures/interpolated-config.valid.json')).toString()
+  );
+
+  // The field used to specify chain ID varies by schema
+  [
+    { testName: 'chains', schema: chainConfigSchema, configObject: { ...config.chains[0] }, chainIdField: 'id' },
+    {
+      testName: 'crossChainRequesterAuthorizer',
+      schema: crossChainRequesterAuthorizerSchema,
+      configObject: { ...config.chains[0].authorizers.crossChainRequesterAuthorizers[0] },
+      chainIdField: 'chainId',
+    },
+  ].forEach((obj) => {
+    const { testName, schema, configObject, chainIdField } = obj;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { contracts, ...objectMissingContracts } = configObject;
+
+    it(`fails if AirnodeRrp contract address within ${testName}.contracts is not specified and there is no deployment for the chain`, () => {
+      const missingChainId = '-5';
+      const chainMissingChainId = {
```

Comment:
The naming feels a bit wierd (saying the chain is missing - I know you mean from deployed references, but the context is not there). I'd just call it `unknownChain` or `nonDeployedChain` or `nonApi3Chain`.

```suggestion
      const unknownChain = {
```